tests/dualtor/test_orch_stress.py

```diff
@@ -0,0 +1,322 @@
+                            'state': 'active'
+                        },
+                    'OP': 'SET'
+                },
+                ...
+            ]
+    """
+    configs = []
+    for intf in mux_intfs:
+        config = {
+            'MUX_CABLE_TABLE:{}'.format(intf):
+                {
+                    'state': state,
+                },
+            'OP': op
+        }
+        configs.append(config)
+    return configs
+
+
+def load_swss_config(dut, config_file):
+    """Load swss config file specified by 'config_file' in swss docker using command 'swssconfig'
+
+    Args:
+        dut (obj): Object for interacting with DUT.
+        config_file (str): Path and filename of the config file in swss docker container.
+    """
+    logger.info('Loading swss config {} ...'.format(config_file))
+    dut.shell('docker exec swss sh -c "swssconfig {}"'.format(config_file))
+    wait(2, 'for CRMs to be updated')
+    logger.info('Loading swss config {} done'.format(config_file))
+
+
+def _swss_path(filename):
+    """Helper function to transform swss config file path.
+
+    Copy files using 'docker cp swss' won't work if the destination path in 'swss' docker is '/tmp'.
+    This helper function is to transform path like '/tmp/filename.json' to '/filename.json'.
+
+    Args:
+        filename (str): Filename
+
+    Returns:
+        str: '/' + basename of filename.
+    """
+    return os.path.join('/', os.path.basename(filename))
+
+
+@pytest.fixture(scope='module')
+def swss_config_files(rand_selected_dut, tor_mux_intfs):
+    """This fixture is to generate/cleanup the swss config files in the swss docker.
+
+    Args:
+        rand_selected_dut (obj): Object for interacting with DUT.
+        tor_mux_intfs (list): List of interface names that are connected to mux cable.
+            Example: ['Ethernet0', 'Ethernet4', ...]
+    """
+    dut = rand_selected_dut
+    swss_configs = [
+        ('active', 'SET', SWSS_MUX_STATE_ACTIVE_CONFIG_FILE),
+        ('standby', 'SET', SWSS_MUX_STATE_STANDBY_CONFIG_FILE),
+        ('active', 'DEL', SWSS_MUX_STATE_DEL_CONFIG_FILE),
+    ]
+    for cfg in swss_configs:
+        mux_config = mux_state_configs(tor_mux_intfs, cfg[0], op=cfg[1])
+        dut.copy(content=json.dumps(mux_config, indent=4), dest=cfg[2])
+        dut.shell('docker cp {} swss:{}'.format(cfg[2], _swss_path(cfg[2])))
+
+    yield
+
+    for cfg in swss_configs:
+        dut.shell('docker exec swss sh -c "rm {}"'.format(_swss_path(cfg[2])))
+        dut.file(path=cfg[2], state='absent')
+
+
+@pytest.fixture(scope='module', autouse=True)
+def config_crm_polling_interval(rand_selected_dut):
+    dut = rand_selected_dut
+    logger.info('Set CRM polling interval to 1 second')
+    dut.shell('crm config polling interval 1')
+    yield
+    logger.info('Set CRM polling interval to 300 seconds')
+    dut.shell('crm config polling interval 300')
+
+
+def compare_crm_facts(left, right):
+    """Compare the crm facts
```

Comment:
Can you please make ```compare_crm_facts``` a common function? I found that several other test cases need to verify crm leaks. Thanks

Reply:
Sure